### ISSUE TITLE
docs: instructions related to NgxMaterialPagesModule are confusing

### ISSUE THREAD

**Issue by Splaktar** [AnthonyNahas/ngx-auth-firebaseui#538]
### Bug Report or Feature Request (mark with an `x`)
```
- [ ] bug report -> please search issues before submitting
- [ ] feature request
- [x] documentation issue
```

### OS and Version?

macOS Catalina

### Versions

```
Angular CLI: 9.1.4
Node: 12.16.1
OS: darwin x64

Angular: 9.1.4
... animations, cli, common, compiler, compiler-cli, core, forms
... language-service, platform-browser, platform-browser-dynamic
... router, service-worker
Ivy Workspace: Yes

Package                           Version
-----------------------------------------------------------
@angular-devkit/architect         0.901.4
@angular-devkit/build-angular     0.901.4
@angular-devkit/build-optimizer   0.901.4
@angular-devkit/build-webpack     0.901.4
@angular-devkit/core              9.1.4
@angular-devkit/schematics        9.1.4
@angular/cdk                      9.2.2
@angular/fire                     6.0.0
@angular/flex-layout              9.0.0-beta.29
@angular/material                 9.2.2
@ngtools/webpack                  9.1.4
@schematics/angular               9.1.4
@schematics/update                0.901.4
rxjs                              6.5.5
typescript                        3.8.3
webpack                           4.42.0
```

### Repro steps
1. Open https://ngx-auth-firebaseui.firebaseapp.com/getting-started
1. Go to Step 6 - Installation
1. Scroll to the bottom

![Screen Shot 2020-05-07 at 22 51 40](https://user-images.githubusercontent.com/3506071/81365330-cb268100-90b5-11ea-9ae9-466d7ba80c74.png)

### The log given by the failure
N/A

### Desired functionality
It says

> Other modules in your application can simply import NgxMaterialPagesModule:

But then the code example after that doesn't appear to have `NgxMaterialPagesModule` in it?

**Comment by AnthonyNahas** [comment]
@Splaktar You have right!

I am building right now a brand new demo app 
<img width="1677" alt="Bildschirmfoto 2020-05-09 um 20 48 05" src="https://user-images.githubusercontent.com/20149774/81482370-cc32ec00-9236-11ea-81a8-df067044727f.png">

I am actually not able to update the old one ...

**Comment by Splaktar** [comment]
I ran the demo app in `src/` and I was surprised that I couldn't actually test or debug the library in the app since it wasn't set up for Firebase Auth. 

Glad to hear that it's still in progress, but the contribution process and maintainability of this library seem to need a little attention atm. I was able to get most of it figured out, and debug in my other app via copy/pasting the build output to my `node_modules/`, but many people would have gotten stuck.

It looks like my fix for CircleCI worked, but now it's just opened up "the next problem" 😁 

My time is limited (like yours I'm sure), but I'm glad that I could help and I hope to be able to assist a little more in the future. If I find some more time, I'd like to resolve the lint/test issues so that I could be more confident of contributions.

It would be great to also `ng add @schuchard/prettier` 😉
